components/embed_text/tests/embed_text_test.py

```diff
@@ -29,15 +29,16 @@ def test_run_component_test():
 
     dataframe = pd.concat({"text": pd.DataFrame(data)}, axis=1, names=["text", "data"])
 
-    component = GenerateEmbeddingsComponent(
+    component = EmbedTextComponent(
```

Comment:
Probably not ideal that we're loading an actual model (which makes the test dependent on an internet connection), although I don't mind as much in the component tests since we currently run them manually